src/daemon/bitcoind/interface.rs

```diff
@@ -432,45 +468,43 @@ impl BitcoinD {
             let amount = utxo
                 .get("amount")
                 .ok_or_else(|| {
-                    BitcoindError(
+                    BitcoindError::Custom(
                         "API break, 'listunspent' entry didn't contain an 'amount'.".to_string(),
                     )
                 })?
                 .as_f64()
                 .ok_or_else(|| {
-                    BitcoindError(
+                    BitcoindError::Custom(
                         "API break, 'listunspent' entry didn't contain a valid 'amount'."
                             .to_string(),
                     )
                 })?;
             let value = Amount::from_btc(amount)
                 .map_err(|e| {
-                    BitcoindError(format!(
+                    BitcoindError::Custom(format!(
                         "Could not convert 'listunspent' entry's 'amount' to an Amount: {}",
                         e.to_string()
                     ))
                 })?
                 .as_sat();
 
-            log::trace!(
-                "Got a new deposit at {:#?} for address {} ({} sats)",
-                &outpoint,
-                &address,
-                &value
-            );
-            new_utxos.insert(
+            new_deposits.insert(
                 outpoint,
                 CachedVault {
                     txo: TxOut {
                         value,
                         script_pubkey,
                     },
-                    status: VaultStatus::Funded,
+                    status: if confirmations < 6 {
```

Comment:
6 is reused line 440: time to make it a constant

Reply:
Right, good catch

Reply:
Done